main.py

```
@@ -1,11 +1,35 @@
 
 def process_query(message, history):
-    messages = []
+    context = []
+    
+    for msg in history:
+        if msg['role'] == 'user' :
+            context.append(HumanMessage(content= msg['content']))
+        else :
+            context.append(AIMessage(content = msg['content'] ))
+    
+    chain = prompt | llm
```

Ruff (0.12.2)

Local variable `chain` is assigned to but never used

Remove assignment to unused variable `chain`

(F841)